### ISSUE TITLE
Jobs Quests Not Showing Up After Daily Reset

### ISSUE THREAD

**Issue by Ryukenstein** [Zrips/Jobs#392]
If i Restart the server Quests they Show up and all **OK**, i can do them and everything is working fine. but if the daily reset at 4am triggers, all quests gets deleted.

And If i do **/jobs resetquest** Says no quests found. i can't get my quests to show up again..., Also if i Restart and i get the quests and then use **/jobs resetquest**, They get deleted again, saying ''no quests found'', no errors at the console at all.

Console says that all quests were succesfully loaded for all my jobs. The only way to solve this is restarting, neither /jobs resetquest nor /jobs reload works. Neither the auto daily reset.

I tested on empty server with only Essentials and Vault. and it Happens. im using

 Spigot 1.13.2 last build tool version with jobs reborn 4.10.3 Help please.

**Comment by Sandukhan** [comment]
I got the same issue with Jobs4.10.3.
I didnt check the new Jobs 4.11.1, but couldnt find anything about this in the changelogs eather. So i quess this may be still a problem. I scheduled the quest reset as such:

  ResetTime:
    Hour: 23
    Minute: 30
However this does not work, or at least not every time. No new quests are picked. Couldnt find any errors in the logs and only restarting the server resets the quests again, wich also happend every restart, wanted or not.
